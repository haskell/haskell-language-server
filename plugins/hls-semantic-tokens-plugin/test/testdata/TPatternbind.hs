module TVariable where

(a, b) = (1, 2)

f x y = x + y




