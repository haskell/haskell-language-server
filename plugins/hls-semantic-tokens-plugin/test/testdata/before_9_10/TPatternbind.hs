module TVariable where

(a, b) = (1, 2)

f g y = g y




