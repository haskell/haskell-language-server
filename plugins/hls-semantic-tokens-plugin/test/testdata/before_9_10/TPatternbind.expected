3:2-3 TVariable "a"
3:5-6 TVariable "b"
5:1-2 TFunction "f"
5:3-4 TFunction "g"
5:5-6 TVariable "y"
5:9-10 TFunction "g"
5:11-12 TVariable "y"
